.github/workflows/golangci-lint.yml

```diff
@@ -6,7 +6,13 @@ on:
       - auto
       - try
       - rollup
+    paths:
```

Comment:
We could isolate this to only go file changes

Reply:
👍 updated